### PR TITLE
Updated ReadMe WRT Hyperion 

### PR DESCRIPTION
**Pull request by luharias**
# Description

Added Hyperion in ReadMe

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,6 +57,14 @@ Devtron is a Web-Based CI/CD Platform for Kubernetes. It integrates various Open
 
 <p align="center"><img src="./assets/readme-comic.png"></p>
 
+<b> Devtron is built in a modular approach. These modules can be installed independently: </b>
+- [Full Devtron Experience](https://github.com/devtron-labs/devtron#tada-Full-Devtron-Experience) - This option gives you all the features of Devtron as a Full Experience providing you with CI, CD, security, cost, observability, stabilization. All the modules stated below are included here.
```
Comment (pghildiyal) [inline README.md:61]: is it ok to call it full devtron experience? it seems like other has some capabilities missing... shouldnt we position as two products? may be we can have section on when to use which

File: README.md [hunk 2]
```diff
@@ -170,14 +175,105 @@ For admin login, use the username:`admin`. And for the password, run the followi
...
+### Current build: 
+
+- Devtron uses modified version of [argo rollout](https://argoproj.github.io/argo-rollouts/)
+- Application metrics only works for k8s 1.16+
+
+
+## 🦹 Hyperion
+
+<details>
+ <summary> <b> Hyperion is one of devtron's Web-based modules to manage helm apps that can be installed seperately too. It helps you observe, manage and debug the applications deployed through Helm across multiple clusters minimizing Kubernetes Complexities. Please expand this column to find Hyperion's features and to get Started with it:</b></summary>
```
Comment (pghildiyal) [inline README.md:189]: devtron should be Devtron

File: README.md [hunk 3]
```diff
@@ -170,14 +175,105 @@ For admin login, use the username:`admin`. And for the password, run the followi
...
+<br>
+
+
+## :tada: Features(For Hyperion)
+ 
+<details><summary> <b> Application-level resource grouping for easier Debugging </b></summary>
+<br>
+
+- Hyperion groups your deployed microservices and displays them in a slick UI for easier monitoring or debugging. Access pod logs and resource manifests right from the Hyperion UI and even edit them!
```
Comment (pghildiyal) [inline README.md:198]: its not microservices, its deployed kubernetes components

File: README.md [hunk 4]
```diff
@@ -170,14 +175,105 @@ For admin login, use the username:`admin`. And for the password, run the followi
...
+
+</details>
+ 
+<details><summary> <b>  Centralized Access Management </b></summary>
+<br>
+ 
+- Give access to users on Project, Environment and App level and control the level of access with customizable View only and Edit access.
```
Comment (pghildiyal) [inline README.md:205]: sentence construction is to be relooked.. word access used twice creates confusion

File: README.md [hunk 5]
```diff
@@ -170,14 +175,105 @@ For admin login, use the username:`admin`. And for the password, run the followi
...
+</details>
+ 
+<details><summary> <b>  Manage and observe Multiple Clusters </b></summary>
+<br>
+ 
+- Manage access of all the Kubernetes clusters (hosted on multiple cloud/on-prem) right from a single Hyperion setup.
```
Comment (pghildiyal) [inline README.md:211]: we are not managing access of kubernetes clusters we are managing access of users to helm charts and applications

Comment (pghildiyal) [reply README.md:211]: yes it is multi-cluster

File: README.md [hunk 6]
```diff
@@ -170,14 +175,105 @@ For admin login, use the username:`admin`. And for the password, run the followi
...
+<details><summary> <b> View and Edit Kubernetes Manifests </b></summary>
+<br>
+ 
+ - View and Edit all Kubernetes resources right from the Hyperion dashboard.
+
+</details>
+
+#### Side Note:
+
+Hyperion module is also a great way to understand how Devtron can fit into your workflow without actually installing apps through Devtron. You can always switch to Full Devtron Experience for all the features. [Just a Couple of Commands away.](https://github.com/devtron-labs/devtron#rocket-getting-started)
```
Comment (pghildiyal) [inline README.md:224]: Hyperion module is also a great way to understand how Devtron can fit into your workflow without actually installing apps through Devtron. -- not true.. person is still not experiencing this

File: README.md [hunk 7]
```diff
@@ -170,14 +175,105 @@ For admin login, use the username:`admin`. And for the password, run the followi
...
+```
+
+You will get result something like below
+
+```text
+[test2@server ~]$ kubectl get svc -n devtroncd devtron-service -o jsonpath='{.status.loadBalancer.ingress}'
+[map[hostname:aaff16e9760594a92afa0140dbfd99f7-305259315.us-east-1.elb.amazonaws.com]]
+```
+
+The hostname mentioned here \( aaff16e9760594a92afa0140dbfd99f7-305259315.us-east-1.elb.amazonaws.com \) is the Loadbalancer URL where you can access the Devtron dashboard.
```
Comment (pghildiyal) [inline README.md:260]: onfuscate url

File: README.md (hunk 3 above)
Comment (luharias) [reply README.md:198]: Changing it to "Deployed Helm Charts"

File: README.md (hunk 5 above)
Comment (luharias) [reply README.md:212]: Making it "Manage Helm charts, Applications across multiple Kubernetes clusters (hosted on multiple cloud/on-prem) right from a single Hyperion setup."

File: README.md (hunk 6 above)
Comment (luharias) [reply README.md:225]: Changing it to "Hyperion module is also a great way to get to know Devtron's UI and some of its features. You can always switch from Hyperion to Devtron which includes all the features."

Let me know if this works, Prashant?

**Comment by luharias** [comment]
Made the changes mentioned

**Review by pghildiyal — DISMISSED** [review]

**Pull request merged**